### PR TITLE
cgp 0143 - Biconomy Account abstraction proposal

### PR DESCRIPTION
**Pull request by himanshugarg06**
https://forum.celo.org/t/final-proposal-to-enable-account-abstraction-erc-4337-on-celo-s-upcoming-l2-leveraging-biconomy-s-aa-infrastructure/8266

### CODE REVIEW

**Comment by work-0xj4an** [comment]
@himanshugarg06 please ensure to adjust your proposal to the [template](https://github.com/celo-org/governance/blob/main/CGPs/cgp-template.md)
Also please be sure to include this info
image

And remember to add to this PR the Folder and the json file.

**Comment by himanshugarg06** [comment]
> @himanshugarg06 please ensure to adjust your proposal to the [template](https://github.com/celo-org/governance/blob/main/CGPs/cgp-template.md) Also please be sure to include this info image
> 
> ‼️ And remember to add to this PR the Folder and the json file.

Done

**Comment by work-0xj4an** [comment]
Please as mentioned before include the transaction description section in the .md file

File: CGPs/cgp-0143.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+ 
+## Overview
+
+This proposal aims to accelerate the growth and long-term adoption of the Celo ecosystem by strategically allocating CELO from the Community Fund toward the deployment of Biconomy’s Account Abstraction infrastructure on Celo’s upcoming L2, to enable builders to create web2-like UX for their dApps.
+
+In this proposal, we’re asking for 200,000 CELO tokens to cover the integration costs of deploying our whole Account Abstraction stack (including Bundlers, Paymasters, and Smart Accounts) on the chain, and the support & maintenance of the infrastructure in the future.
+
+### Status
+
+- DRAFT = Feedback collection
```
Comment (work-0xj4an) [inline CGPs/cgp-0143.md:20]: Only "DFRAT"

**Review by work-0xj4an — APPROVED** [review]

File: CGPs/cgp-0143/mainnet.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[
+    {
+      "contract": "GoldToken",
+      "address": "0x471EcE3750Da237f93B8E339c536989b8978a438",
+      "function": "increaseAllowance",
```
Comment (0xzoz) [inline CGPs/cgp-0143/mainnet.json:5]: Just wanted to verify that `increaseAllowance` is the correct method to be called? Im guessing it is but other requests for funds usually use `approve`

```
[
    {
    "contract": "GoldToken",
    "function": "approve",
    "args": ["0xda2069f47D252121c2288301D6EF50B87220A693","1600000000000000000000000"],
    "value": "0"
    }
]
```

**Review by 0xzoz — APPROVED** [review]

**Pull request merged**